docs/root/configuration/http/http_filters/grpc_streaming_filter.rst

```diff
@@ -0,0 +1,20 @@
+.. _config_http_filters_grpc_streaming:
+
+gRPC Streaming
```

Comment:
High level comment: can we potentially call this is the "gRPC Statistics" filter or similar? My general feeling is that we should actually replicate all of the stats that we produce in the bridge filter in this filter. It's not intuitive at all to people that they can use the bridge filter to get stats and this would make it a much better experience. I think most of the underlying stats code could be shared either as a utility class or as a common base class between the filters.

Reply:
We use the Envoy's gRPC bridge filter only for these stats. It would make a lot of sense to have this "gRPC statistics" filter.

Reply:
Ok, that sounds reasonable. Do you want to also remove stats code from gRPC bridge?

Reply:
I would probably leave them for now but call them deprecated in the docs and we can remove in a future version. For now I would just share the code and then we can delete it from the bridge filter later.

Reply:
Updated the docs, added deprecation/version history notes. I copied the tests from bridge filter to validate the stats.